### PR TITLE
Extracting images only

### PR DESCRIPTION
**Pull request by michalbubula**

### CODE REVIEW

**Review by cneud — APPROVED** [review]
Thanks @vahidrezanezhad, LGTM for merge now!

**Review by kba — APPROVED** [review]
LGTM

File: src/eynollah/eynollah.py [hunk 1]
```diff
@@ -226,6 +228,7 @@ def __init__(
         self.model_page_dir = dir_models + "/eynollah-page-extraction_20210425"
         self.model_region_dir_p_ens = dir_models + "/eynollah-main-regions-ensembled_20210425"
         self.model_region_dir_p_ens_light = dir_models + "/eynollah-main-regions_20220314"
+        self.model_region_dir_p_ens_light_only_images_extraction = dir_models + "/eynollah-main-regions_20231127_672_org_ens_11_13_16_17_18"
```
Comment (cneud) [inline src/eynollah/eynollah.py:231]: Is a new model introduced here @vahidrezanezhad?

**Review by cneud — CHANGES_REQUESTED** [review]
I think these steps are still required:
1. Update [README.md](https://github.com/qurator-spk/eynollah/blob/extracting_images_only/README.md) to document the added functionality
2. New model `eynollah-main-regions_20231127_672_org_ens_11_13_16_17_18`?
In this case, the model must be published first, model archives and `ocrd-tool.json` updated etc. according to our guidelines.

File: src/eynollah/eynollah.py (hunk 1 above)
Comment (vahidrezanezhad) [reply src/eynollah/eynollah.py:231]: Yes. eynollah-main-regions_20231127_672_org_ens_11_13_16_17_18 is the new model.

Comment (cneud) [reply src/eynollah/eynollah.py:231]: Thanks for publishing the model on HF. 

I created a new [models_eynollah.tar.gz](https://github.com/qurator-spk/eynollah/releases/download/v0.3.1/models_eynollah.tar.gz) release asset for `v0.3.1` that includes the new model and also 
- updated [`ocrd-tool.json`](https://github.com/qurator-spk/eynollah/pull/132/commits/351e9a897a390cc5978346ae56bd725f021876d9) 
- and [`Makefile`](https://github.com/qurator-spk/eynollah/pull/132/commits/327b446a16cc0d28281d41c96de1062c18293601) 

@michalbubula Can you please 

> 1. Update [README.md](https://github.com/qurator-spk/eynollah/blob/extracting_images_only/README.md) to document the added functionality

Comment (michalbubula) [reply src/eynollah/eynollah.py:231]: I updated the Readme and the Wiki.

**Review by cneud — APPROVED** [review]

**Pull request merged**